pkgs/by-name/li/libgit2/package.nix

```diff
@@ -46,7 +46,7 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.cmakeBool "USE_GSSAPI" withGssapi)
     "-DBUILD_SHARED_LIBS=${if staticBuild then "OFF" else "ON"}"
   ]
-  ++ lib.optionals stdenv.hostPlatform.isWindows [
+  ++ lib.optionals (stdenv.hostPlatform.isWindows && !stdenv.hostPlatform.isUnix) [
     "-DDLLTOOL=${stdenv.cc.bintools.targetPrefix}dlltool"
```

Comment:
I think that line should probably be passed unconditionally? Even on non-Windows, it should be harmless, right?